udata/linkchecker/backends.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals, absolute_import
+
+import logging
+import pkg_resources
+from datetime import datetime
+
+log = logging.getLogger(__name__)
+
+DEFAULT_LINKCHECKER = 'croquemort'
```

Comment:
I think that croquemort should not be referenced here but comes from udata-croquemort's settings.

Reply:
Maybe introduce a `DEFAULT_LINKCHECKER` configuration variable with `no_check` by default and has to be set to `croquemort` when you install `udata-croquemort`?

Reply:
Or something like `LINK_CHECKER = 'path.to.DummyChecker'` by default, I kinda like the django-like Python path in settings.

Reply:
I pushed something more coherent IMHO. `no_check` is now an entrypoint provided linkchecker in udata (there is your django-like path ;-)). `LINKCHECKING_DEFAULT_LINKCHECKER` is by default the key to this linkchecker. It will be replaced by `croquemort` if you install `udata-croquemort` and want it to be the default one (or whatever linkchecher you want).